plugins/modules/verify_cip_security.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Red Hat Inc.
+# MIT (see COPYING or https://opensource.org/licenses/MIT)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community",
+}
+
+DOCUMENTATION = """
+---
+module: verify_cip_security
+short_description: Verify CIP Security is available and/or enabled
+description:
+  - Verify CIP Security is available, which security profiles and are they configured
+author:
+  - Matthew Sandoval (@matoval)
+"""
+
+EXAMPLES = """
+- name: Verify CIP Security
+  community.cip.verify_cip_security:
+  register: cip_security
+
+- name: Debug
+  ansible.builtin.debug:
+    var: cip_security
+"""
+
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.community.cip.plugins.module_utils.logix import LogixUtil
+
+
+def main():
+    module = AnsibleModule(
+        argument_spec=dict(),
+        supports_check_mode=True
+    )
+
+    logix_util = LogixUtil(module)
+
+    all_available_attr = logix_util.plc.generic_message(service=logix_util.cip.Services.get_attribute_single, class_code=0x5D, instance=0, attribute=0, data_type=logix_util.cip.UINT)  # noqa yaml[line-length]
+
+    msg = []
+
+    if "Destination unknown" in str(all_available_attr[3]):
+        msg.append("CIP Security is unsupported")
+    else:
+        msg.append("CIP Security is supported")
+
+        profiles = logix_util.plc.generic_message(service=logix_util.cip.Services.get_attribute_single, class_code=0x5D, instance=1, attribute=2, data_type=logix_util.cip.WORD, name='Security Profiles')  # noqa yaml[line-length]
```

Comment:
this (and the following lines) should be reformatted to not need to ignore the linter, [python black](https://github.com/psf/black) will automatically reformat this and make the linter happy

Reply:
I had to leave one ignore in but fix the other ones